.github/workflows/ci-mobile-save-for-later-user-deletion.yaml

```
@@ -0,0 +1,35 @@
+name: CI
+on: push
+
+jobs:
+  mobile-save-for-later-user-deletion-ci:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v3
+      - uses: aws-actions/configure-aws-credentials@v1
```

Might wanna bump this, shouldn't make any noticable difference, but will remove some warnings from older version of AWS javascript sdk which is deprecated
```suggestion
      - uses: aws-actions/configure-aws-credentials@v4
```

---

Likewise for other items below